script.js

```diff
@@ -140,4 +140,47 @@ document.addEventListener('DOMContentLoaded', () => {
       event.run();
     }
   });
+  // Local storage for the form
+  const firstName = document.getElementById('name');
+  const emailAdd = document.getElementById('email');
+  const sms = document.getElementById('msg');
+
+  const storageForm = {
+    fName: '',
+    fEmail: '',
+    fMsg: '',
+  };
+
+  function populateStorage() {
+    storageForm.fName = firstName.value;
+    storageForm.fEmail = emailAdd.value;
+    storageForm.fMsg = sms.value;
+    localStorage.setItem('data', JSON.stringify(storageForm));
+  }
+
+  function getFormStorage() {
+    const reFill = JSON.parse(localStorage.getItem('data'));
+    firstName.value = reFill.fName;
+    emailAdd.value = reFill.email;
+    sms.value = reFill.msg;
+  }
+
+  function setDataFirst() {
+    if (localStorage.getItem('data') === null) {
+      localStorage.setItem('data', JSON.stringify(storageForm));
+    }
+  }
+
+  getFormStorage();
```

Comment:
- [ ] You should run this function if localStorage has data. :thinking: You can use

```
    if (localStorage.getItem('data') !== null) {
      getFormStorage();
    }
```